setup.sh

```diff
@@ -230,13 +265,19 @@ else
 		exit 3
 	fi
 	if [ ! -d $var_dir ]; then
-		mkdir $var_dir
+		mkdir -p $var_dir
 		if [ $? != 0 ]; then
-			echo "ERROR: Failed to create directory $var_dir"
-			echo ""
-			exit 3
+			mkdir $var_dir
```

Comment:
Why is a second mkdir needed?

Reply:
The second `mkdir` is needed as a presumption that some OS may not support the `-p` flag.

Reply:
I checked a few OS man pages, and I couldn't see any that don't support -p.

Reply:
Alright, thank you for checking on that. I have fixed that.